### DISCUSSION TITLE
Error 1101 when invoking an Edge Function

### DISCUSSION THREAD

**Discussion by soleuil** [supabase/supabase#19561] [Questions]
I have created an deployed a pretty simple edge function which is like this:
```
import { serve } from "https://deno.land/std@0.168.0/http/server.ts";

console.log("Hello from Functions!")

serve(async (req) => {
  const payload = await req.json()
  console.log("Hello from Functions!")

  return new Response("ok");
});
```
I created a Webhook to invoke but seeing that there was no log or invocations at all when creating a new row I tested invoking the function from the terminal and from postman and I obtained the same Error 1101.

I would like to know if there is anything else I can do to debug the issue or if it is something out of my scope.

Thanks.
-------------Edit original port------
As I have two projects and in the other one I already have an edge function, I just tried to invoke the function from the other project and it just work fine. So I am having the issues only in one of my projects where I keep getting the Error 1101.
I have also tested to delete the original edge function and create a different one and I keep getting the same Error in that project. I have also restarted the project but after going back online the issue persist.

**Comment by yjgaia** [comment]
I too am experiencing the same problem. The problem is occurring only for newly created projects.

**Comment by GaryAustin1** [comment]
![image](https://github.com/supabase/supabase/assets/54564956/4813183a-f0c3-4d15-a90b-80f53d2861a4)
